docs/machine-learning/tutorials/sentiment-analysis.md

```diff
@@ -151,118 +184,136 @@ Apply a <xref:Microsoft.ML.Transforms.TextFeaturizer> to convert the `SentimentT
 
 [!code-csharp[TextFeaturizer](../../../samples/machine-learning/tutorials/SentimentAnalysis/Program.cs#7 "Add a TextFeaturizer to the pipeline")]
 
-### About the classification model
-
-Classification is a machine learning method that uses data to **determine** the category, type, or class of an item or row of data. For example, you can use classification to:
-
-* Identify sentiment as positive or negative.
-* Classify email filters as spam, junk, or good.
-* Determine whether a patient's lab sample is cancerous.
-* Categorize customers by their propensity to respond to a sales campaign.
-
-Classification tasks are frequently one of the following types:
-
-* Binary: either A or B.
-* Multiclass: multiple categories that can be predicted by using a single model.
+## Choose a learning algorithm
 
 The <xref:Microsoft.ML.Trainers.FastTreeBinaryClassifier> object is a decision tree learner you'll use in this pipeline. Similar to the featurization step, trying out different learners available in ML.NET and changing their parameters leads to different results. For tuning, you can set [hyperparameters](../resources/glossary.md#hyperparameter) like <xref:Microsoft.ML.Trainers.FastTreeBinaryClassifier.NumTrees>, <xref:Microsoft.ML.Trainers.FastTreeBinaryClassifier.NumLeaves>, and <xref:Microsoft.ML.Trainers.FastTreeBinaryClassifier.MinDocumentsInLeafs>. These hyperparameters are set before anything affects the model and are model-specific. They're used to tune the decision tree for performance, so larger values can negatively impact performance.
 
-Add the following code to the `TrainAndPredict` method:
+Add the following code to the `Train` method:
 
 [!code-csharp[BinaryClassifier](../../../samples/machine-learning/tutorials/SentimentAnalysis/Program.cs#8 "Add a fast binary tree classifier")]
 
 ## Train the model
 
 You train the model, <xref:Microsoft.ML.PredictionModel%602>, based on the dataset that has been loaded and transformed. `pipeline.Train<SentimentData, SentimentPrediction>()` trains the pipeline (loads the data, trains the featurizer and learner). The experiment is not executed until this happens.
 
-Add the following code to the `TrainAndPredict` method:
+Add the following code to the `Train` method:
 
 [!code-csharp[TrainModel](../../../samples/machine-learning/tutorials/SentimentAnalysis/Program.cs#9 "Train the model")]
 
-## Predict the test data outcomes with the model
+### Save and Return the model trained to use for evaluation
 
-Add some comments to test the trained model's predictions in the `TrainAndPredict` method:
+At this point, you have a model that can be integrated into any of your existing or new .NET applications, or continue to work with it. For this tutorial, save your model to a .zip file before returning, so add the following code to the next line in `Train`:
```

Comment:
last sentence has a weird flow. Maybe break it down or connect the two sentences better?

Reply:
Yeah, I struggled with this one.